administrator/language/de-DE/com_csp.ini

```
@@ -24,8 +24,11 @@ COM_CSP_CONTENTSECURITYPOLICY_NONCE_ENABLED="<a href='https://scotthelme.co.uk/c
 COM_CSP_CONTENTSECURITYPOLICY_NONCE_ENABLED_DESC="Einen kryptografische Nonce für alle inline-skripte und inline-styles generieren, welche über die Joomla-API in die Seite geladen werden, und mittels Content-Security-Policy freigeben. Die Angabe der Nonce führt dazu, dass ein moderner Browser 'unsafe-inline' ignoriert."
 ; Please do not translate 'Content-Security-Policy' & 'Content-Security-Policy-Report-Only' in the following language string
 COM_CSP_CONTENTSECURITYPOLICY_REPORT_ONLY_DESC="Den Header 'Content-Security-Policy-Report-Only' anstelle von 'Content-Security-Policy' verwenden."
-; Please do not translate the following language string
+; Please do not translate the following two language strings
 COM_CSP_CONTENTSECURITYPOLICY_REPORT_ONLY="Report-Only"
+COM_CSP_CONTENTSECURITYPOLICY_STRICT_DYNAMIC_ENABLED="strict-dynamic"
+; Please do not translate 'strict-dynamic', 'self' and 'unsafe-inline' in the following language string
+COM_CSP_CONTENTSECURITYPOLICY_STRICT_DYNAMIC_ENABLED_DESC="Der Ausdruck 'strict-dynamic' legt fest, dass einem im HTML vorhandenes Skript explizit vertraut wird, wenn sie eine nonce oder einen Hash haben. Die Angabe von 'strict-dynamic' führt dazu, dass ein moderner Browser alle explizit erlaubten Domains sowie anderen Ausdrücke wie 'self' oder 'unsafe-inline' ignoriert."
```

```suggestion
COM_CSP_CONTENTSECURITYPOLICY_STRICT_DYNAMIC_ENABLED_DESC="Der Ausdruck 'strict-dynamic' legt fest, dass einem im HTML vorhandenes Skript explizit vertraut wird, wenn sie eine Nonce oder einen Hash haben. Die Angabe von 'strict-dynamic' führt dazu, dass ein moderner Browser alle explizit erlaubten Domains sowie anderen Ausdrücke wie 'self' oder 'unsafe-inline' ignoriert."
```

Nonce statt nonce?
in Zeile 24 haben wir auch Nonce